.github/workflows/digester_bot.yml

```diff
@@ -8,15 +8,26 @@ jobs:
     if: (github.repository == 'kubevirt/hyperconverged-cluster-operator')
     runs-on: ubuntu-latest
     steps:
+      - name: check for existing PR
+        run: |-
+          if curl -s -L  https://api.github.com/repos/kubevirt/hyperconverged-cluster-operator/pulls | jq .[].title | grep -q "Update Image Digests"; then
```

Comment:
Can we use a constant (environment variable?) with the title of the PR, "Update Image Digests"? Both when setting and when grepping... to avoid rename-bugs.